### ISSUE TITLE
Machine Learning 

### ISSUE THREAD

**Issue by BeastlyPixels** [freqtrade/freqtrade#3439]
## Describe your environment

  * Operating system: _Raspian_
  * Python Version: __3___ (`python -V`)
  * CCXT version: _____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: __standard__ (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)
  
## Your question

Would hyperopt take advantage of an Intel Neural compute stick 2 plugged into my pi?
https://thenewstack.io/a-closer-look-at-intel-movidius-neural-compute-stick/

**Comment by xmatthias** [comment]
hyperopt utilizes [scikit-optimize](https://scikit-optimize.github.io/stable/) - which under the hood uses [scikit-learn](https://scikit-learn.org/).

Knowing that they don't support GPU (and don't intend to support GPU in the near future to keep installation of their modules simple) - i doubt they will support this, but don't know for sure as i'm not involved in these projects.
However, i'd suggest to ask in the appropriate support channels for either of these projects.

**Comment by hroff-1902** [comment]
pls also have a look at: https://github.com/freqtrade/freqtrade/issues/2754#issuecomment-571722253

(citing scikit-learn docs:)
> No, or at least not in the near future. The main reason is that GPU support will introduce many software dependencies and introduce platform specific issues. scikit-learn is designed to be easy to install on a wide variety of platforms. Outside of neural networks, GPUs don’t play a large role in machine learning today, and much larger gains in speed can often be achieved by a careful choice of algorithms.

I would close it as a duplicate.

**Comment by xmatthias** [comment]
yeah, it's kindof a duplicate.

If you find out if it works would be great if you could comment below, thanks.

**Comment by BeastlyPixels** [comment]
Okay I will reach out to the hyperopt team and post my findings. Thank you!

**Comment by hroff-1902** [comment]
you may ask the scikit-learn team if they would support Intel sticks...
